service/etcd/etcdClient.go

```diff
@@ -26,27 +25,19 @@ type EtcdClient struct {
 	leaseResponse *clientv3.LeaseGrantResponse
 }
 
-//func NewEtcdClient() *clientv3.Client {
 func NewEtcdClient() *EtcdClient {
-	// TODO: Need to fix as constant variables
-	certFile := "/etc/kubernetes/pki/etcd/server.crt"
-	keyFile := "/etc/kubernetes/pki/etcd/server.key"
-	caFile := "/etc/kubernetes/pki/etcd/ca.crt"
-	endPoints := "https://10.0.2.15:2379"
```

Comment:
Kindly read all the cert location and EtcdEndPoints value from config. As this cert location and IP might vary based on clusters. Different clusters might have different cert locations. Suggest to read the value from kvmsservice.yaml.